debug/test/browser/component-stack.test.js

```diff
@@ -79,7 +79,7 @@ describe('component stack', () => {
 		render(<Bar />, scratch);
 
 		let lines = getStack(errors).split('\n');
-		expect(lines[0].indexOf('td') > -1).to.equal(true);
+		expect(lines[0].indexOf('tr') > -1).to.equal(true);
```

Comment:
This generates two warnings, which are both correct imo. The difference is that the order is only swapped because we do the validation based on the order in `options.diffed` which is the reverse of what we had before.